tests/Features/Skip.php

```diff
@@ -31,3 +35,19 @@
 it('skips when skip after assertion')
     ->assertTrue(true)
     ->skip();
+
+it('skips if the PHP version is the current one (with message)')
+    ->skipForVersion(PHP_VERSION, null, 'because of the current version')
+    ->assertTrue(true);
+
+it('skips if the PHP version is greater that 7 (should skip)')
+    ->skipForVersion('7', '>')
```

Comment:
I think would be cool if we can use this API Like so:

```
->skipForVersion('7.0.1', 'reason'); // specific version
->skipForVersion('^7.0', 'reason'); // any version bigger than 7 but lower than 8, like composer.
```

Reply:
I had originally thought about that, but we'd probably want to pull in another package such as [`composer/semver`](https://github.com/composer/semver). 🤷🏻

Then it would be:
```php
Composer\Semver\Semver::satisfies($version, $constraint);
```